src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -18,8 +20,13 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	public void actionPerformed(ActionEvent e) {
 		if(e.getActionCommand().equals("Figure Joe 1")) {
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-		}else
-		{
+		}else if(e.getActionCommand().equals("Square")){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.